lib/puppet/functions/fqdn_rotate.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# @summary DEPRECATED.  Use the namespaced function [`stdlib::fqdn_rotate`](#stdlibfqdn_rotate) instead.
```

Comment:
```suggestion
# @summary Rotates an array or string a random number of times, combining the `$fqdn` fact
#    and an optional seed for repeatable randomness.
# @deprecated Use the namespaced function [`stdlib::fqdn_rotate`](#stdlibfqdn_rotate) instead.
```

deprecated should be supported in puppet-strings 4.0.0 which we should be able to use once the bolt dependency is update (https://github.com/puppetlabs/bolt/pull/3204)